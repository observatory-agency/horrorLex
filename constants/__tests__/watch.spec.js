const watch = require('../watch');

describe('watch', () => {
  it('should match the snapshot', () => {
    expect(watch).toMatchSnapshot();
  });
});
