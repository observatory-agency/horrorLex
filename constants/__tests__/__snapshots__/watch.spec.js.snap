// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`watch should match the snapshot 1`] = `
Array [
  "./views",
]
`;
